pages/docs/concepts/asyncapi-document/structure.md

```diff
@@ -0,0 +1,432 @@
+    name: Rohit
+    email: rohit@asyncapi.com
+  license:
+    name: Apache 2.0
+    url: https://www.apache.org/licenses/LICENSE-2.0.html
+  tags:
+    - name: Events
+      description: APIs related to event streaming
+    - name: Authentication
+      description: APIs for authentication and authorization
+  externalDocs:
+    description: Additional documentation 
+    url: https://example.com/docs
+```
+
+### `Servers` object
+The `servers` object in an AsyncAPI document is a map of `server` objects that defines the network endpoints or brokers that applications can connect to for exchanging messages. It specifies the details necessary to establish a connection, such as the protocol, host, port, and additional connection options. The `server` object is used to capture details such as URIs, protocols and security configuration. 
+
+The purpose of the `servers` object is to provide the necessary information for clients to connect to the message broker or server and participate in the message exchange or listen to the events. By defining multiple servers, the AsyncAPI document can accommodate different environments or deployment scenarios, such as production, staging, or development.
+
+The `servers` object typically includes the following properties(few are mandatory, few are optional) for each server:
+
+- `host`: The server host name. It may include the port. This field supports Server Variables. 
+- `protocol`: The protocol or messaging protocol used by the server (e.g., AMQP, MQTT, WebSocket).
+- `protocolVersion`: The version of the protocol used for connection.
+- `pathname`: The path to a resource in the host.
+- `description`: An optional string describing the server.
+- `title`: A human-friendly title for the server.
+- `summary`: A short summary of the server.
+- `security`: A declaration of which security schemes can be used with this server. 
+- `tags`: A list of tags for logical grouping and categorization of servers.
+- `externalDocs`: Additional external documentation for this server.
+- `bindings`: A map where the keys describe the name of the protocol and the values describe protocol-specific definitions for the server.
+
+Here's a visual representation of the `server` object and its properties:
+```mermaid
+graph LR
+  A[server]
+  B(host)
+  C(pathname)
+  D(protocol)
+  E(description)
+
+  A --> B
+  A --> C
+  A --> D
+  A --> E
+```
+
+Here's a code example of the servers object with multiple servers in an AsyncAPI document:
+```yaml
+host: rabbitmq.in.mycompany.com:5672
+pathname: /production
+protocol: amqp
+description: Production RabbitMQ broker (uses the `production` vhost).
+```
+
+### `Channels` object
+The `channels` object in an AsyncAPI document hold the relative paths to the individual channels and their operations. Channel paths are relative to servers. The `channels` represents the communication pathways through which messages are exchanged. The `channel` object sescribes a shared communication channel. 
```

Comment:
```suggestion
The `channels` object in an AsyncAPI document hold the relative paths to the individual channels. Channel paths are relative to servers. The `channels` represents the communication pathways through which messages are exchanged. The `channel` object describes a shared communication channel. 
```

Not anymore in v3. Later below we describe the `operations` object so we can skip it here.